run_local_simple.py

```
@@ -0,0 +1,132 @@
+from pathlib import Path
+
+sys.path.append(str(Path(__file__).parent))
+
+# Test imports
+print("Testing imports...")
+try:
+    from src.rss.filing_tracker import FilingTracker
+    from src.rss.petition_processor import Chapter11PetitionProcessor
+    from src.pacer.models import FilingType
+    print("✅ Imports successful")
+except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)